packages/vapor/scss/components/navigation.scss

```diff
@@ -11,7 +11,8 @@
 }
 
 .navigation {
-    width: $navigation-width;
+    min-width: $navigation-width;
+    max-width: $navigation-width;
```

Comment:
What does it change to do this ?

Reply:
Somehow the width of the side nav was changing when clicking on smaller items. I don't really understand why, but by setting a min and a max width instead the width of the side nav always stay the same :taco: